packages/TorneloScoresheet/src/components/Toolbar/Toolbar.tsx

```diff
@@ -62,6 +68,28 @@ const Toolbar: React.FC = () => {
   const currentColour = colourForMode[appModeState.mode];
   const arbiterModeVisibility = arbiterModeDisplay[appModeState.mode];
   const currentTextColour = textColour(currentColour);
+
+  const graphicalRecordingState = useGraphicalRecordingState();
+  const goToArbiterGameFromGraphicalRecording =
+    graphicalRecordingState?.[1]?.goToArbiterGameMode;
+
+  const tablePairingState = useTablePairingState();
+  const goToArbiterGameFromTablePairing =
+    tablePairingState?.[1]?.goToArbiterGameMode;
+
+  const handleVerify = () => {
+    if (appModeState.mode == AppMode.GraphicalRecording) {
+      if (!goToArbiterGameFromGraphicalRecording) {
+        return;
+      }
+      goToArbiterGameFromGraphicalRecording();
+    } else {
+      if (!goToArbiterGameFromTablePairing) {
+        return;
+      }
+      goToArbiterGameFromTablePairing();
+    }
+  };
```

Comment:
This will get messy as we add more and more AppModes here, maybe we should store a dictionary mapping the game mode to the transition function?

ie at the top:
  `const appModeToAribiterMode = Record<AppMode, () => void>`
and then in handleVerify() :
 `appModeToArbiterMode[appModeState]()`

also if we change the appMode to ArbiterGameMode, we will loose all the data in the PlayerMode states, so either we need to pass all the data to the ArbiterGameMode or we need to find a way to store it, I think we need to discuss this all together to decide what the best way of handling the arbiter mode will be, maybe we should have a separate arbiter mode per app mode ie have a TablePairingArbiterMode, GraphicalRecordingArbiterMode, ... etc

Reply:
I couldn't get the dictionary to work but I turned it into a switch statement so it's a bit neater. We would probably have a maximum of 4 possible changes to arbiter mode (tablePairing, graphicalrecording, resultdisplay, and textrecording) so hopefully it shouldn't offend Harry too much